canonical-websites/jp.ubuntu.com
# Update to Vanilla 1.6.5 + minor styling fixes to homepage

## Done

- Updated to Vanilla 1.6.5
- Small styling fixes

## QA

- `./run`
- Navigate to http://0.0.0.0:8012/
- Check that site is now running Vanilla 1.6.5
- Check that there are no styling errors

## Issue / Card

Fixes vanilla-framework/vanilla-framework#1468
